packages/use-i18n/src/__tests__/__snapshots__/formatDate.ts.snap

```diff
@@ -24,45 +24,45 @@ exports[`formatDate should work with custom format and locale es 2`] = `"jueves,
 
 exports[`formatDate should work with custom format and locale es 3`] = `"jueves, 13 F 20 d. C., 15:28:00 (tiempo universal coordinado)"`;
 
-exports[`formatDate should work with custom format and locale fr 1`] = `"jeudi 13 F 20 ap. J.-C., 16:28:00 temps universel coordonné"`;
+exports[`formatDate should work with custom format and locale fr 1`] = `"jeudi 13 F 20 ap. J.-C. 16:28:00 temps universel coordonné"`;
```

Comment:
Why ? 🤔

Reply:
node 18...

Reply:
But this should be updated in this PR ? @Lawndlwd

Reply:
i did` pnpm test` it fails on almost all packages, so it seems it needed snapshot update

Reply:
what i need to do more than `rm -rf node module && pnpm install` to update my pnpm-lock ?